### PR TITLE
Issue and PR templates

### PR DESCRIPTION
**Pull request by bruce**
We're getting to the volume now that it would be helpful to:

For Issues:
- Direct people to better sources of information for questions and support
- Remind them another package may be a better place to start
- Collect helpful environmental and schema information for bug reports

For PRs:
- Indicate helpful things to include that will shorten the review process, including tests, specs, and docs
- Remind them to match style
- Other things?

It should be simple, friendly, and quick to navigate

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+-->
+
+<!-- START BUG TEMPLATE (Please delete the rest of this if you're not filing a bug.) -->
+
+If submitting a bug, please provide the following:
+
+### Environment
+
+* Elixir version (elixir -v):
+* Absinthe version (mix deps; please list all Absinthe-related packages):
```
Comment (benwilson512) [inline .github/issue_template.md]: I like doing `mix deps |grep absinthe` here.

Comment (bruce) [reply .github/issue_template.md]: Updated.

**Comment by benwilson512** [comment]
:shipit:

**Comment by bruce** [comment]
Going to go ahead and get this in 🚀

**Pull request merged**